game/Bird.java

```diff
@@ -124,11 +124,11 @@ public void update() {
         if (Resources.IN_TUBE) {
             int leftBird = Resources.BIRD_X_POSITION;
             int rightBird = Resources.BIRD_X_POSITION + Resources.BIRD_WIDTH;
-            int leftTube = (int) Resources.CURRENT_TUBE.getX();
-            int rightTube = (int) Resources.CURRENT_TUBE.getX() + Resources.TUBE_WIDTH;
+            int leftTube = (int) Resources.CURRENT_TUBE.getX()+50;
```

Comment:
Ah, I meant the original, this fixes the problem :+1: got a gen 2 bird to go forever with your version.